python/packages/ag-ui/agent_framework_ag_ui/_message_adapters.py

```diff
@@ -242,6 +242,19 @@ def agent_framework_messages_to_agui(messages: list[ChatMessage] | list[dict[str
                     import json
 
                     content_text = json.dumps(content.result)  # type: ignore
+                elif isinstance(content.result, list):
+                    # Handle Contents list (e.g., from MCP tool results)
+                    import json
```

Comment:
we should move the `import json` to the top of the file - it appears twice now (once on line 242 and once here).